### ISSUE TITLE
minScale is not respected when contain: 'outside'

### ISSUE THREAD

**Issue by roonie007** [timmywil/panzoom#604]
### IMPORTANT: 
- I only have the bug when I import the module as the following: `import panzoom from '@panzoom/panzoom';`
- when I tried to reproduce the bug using **<script>** + **CDN link** it worked as expected ( you can check it [here](https://jsbin.com/votiqoxoyi/edit?html,css,js,output) ).

### Describe the bug
I have a canvas with the size 10000x10000, when using `contain: 'outside'`, the `minScale` option is not respected.

if i remove the `contain: 'outside'` the `minScale` is respected correctly.

### Your environment

- I am using vue ( I copy passed my component on the reproduction link below)
- Version of panzoom: 4.4.3
- Browser and browser version: Chrome 97.0.4692.71 

### Expected behavior

`minScale` should be taken in account and zooming out should not go beyond `minScale`

### Actual behavior

If you set something like this: 

```javascript
const instance = panzoom(canvas, {
    minScale: 1, // any value here is not respected when contain equal 'outside'
    maxScale: 2,
    contain: 'outside',
};
```
You can always zoom out to **0.1823**.

The `minScale` is not respected, but it `maxScale` is respected

### Steps to reproduce

https://codesandbox.io/s/stoic-gates-nhpyq?file=/src/App.vue

**Comment by timmywil** [comment]
Thanks for opening an issue. My guess is it's working with the script tag not because the code is somehow different but because the script is loading at a different time. You could get different results depending on whether the CSS had been rendered or not when Panzoom is initialized. 4.4.3 had a bug where if the element's size ever changed, the dimensions for contain would be off. That is fixed on the main branch and I just released 4.4.4. I tried 4.4.4 in your sandbox and it seems to work now.

**Comment by roonie007** [comment]
Thanks, I can confirm bumping to 4.4.4 everything works perfectly.
